sky/execution.py

```diff
@@ -56,7 +57,9 @@ def _maybe_clone_disk_from_cluster(clone_disk_from: Optional[str],
                                 f'{clone_disk_from!r}'):
         image_id = original_cloud.create_image_from_cluster(
             clone_disk_from,
```

Comment:
This argument should be removed?
```suggestion
```

Reply:
Sorry about this. I've missed some basic errors. It should be fixed now.

Reply:
It looks like `clone_disk_from` should be the display name in this case?

Reply:
Yes, the `clone_disk_from` should be the same as `handle.cluster_name` as we retrieved the handle with `clone_disk_from`. : )